index.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 var chalk = require('chalk');
+var os = require('os');
+var support = process.platform !== 'win32' || process.versions.electron || parseFloat(os.release()) >= 10;
```

Comment:
support => isSupported

parseFloat is not a good way to compare versions. They're not numbers. Instead use `String#split()` and only compare the first part.

For clarity `parseFloat(os.release()) >= 10` => `(process.platform === 'win32' && parseFloat(os.release()) >= 10)`.